WordPress/WordPressTest/RemoteFeatureFlagTests.swift

```diff
@@ -66,6 +69,54 @@ class RemoteFeatureFlagTests: XCTestCase {
         XCTAssertFalse(store.value(for: MockFeatureFlag.remotelyDisabledLocallyEnabledFeature))
         XCTAssertFalse(store.value(for: MockFeatureFlag.remotelyDisabledLocallyDisabledFeature))
     }
+
+    func testThatUpdateIfNeededDoesNotUpdateIfCacheIsValid() {
+        // Given
+        let userDefaults = UserPersistentStoreFactory.instance()
```

Comment:
Given the suggestion above for injectable defaults, you could inject these defaults into the `MockFeatureFlagRemote`.